src/test/resources/com/znsio/teswiz/features/weatherAPI.feature

```diff
@@ -0,0 +1,30 @@
+@weatherAPI @api
+Feature: Weather data based on location Coordinates
+# CONFIG=./configs/api_local_config.properties TAG=weatherAPI PLATFORM=api ./gradlew run
+
+  @temperature
```

Comment:
separate workflow tests in a different package.

Reply:
updated